geometry_msgs/Point position3d
geometry_msgs/Point positionPixel